assin.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+<?php
+$f_name =  $_POST["fname"];       //store first name via post method
+$l_name =  $_POST["lname"];       //store last name via post method
+$full_name = $f_name." ".$l_name; //concatenate the fname and lname
+
+?>
+
+
+Hello <?php echo $full_name;?>   <!-- print the full name in web page -->
```

Comment:
Do we need to close php tag above? Can't we do this without closing it.

Reply:
Yes, we can do this without closing the first php code.